liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java

```diff
@@ -58,7 +74,8 @@ public void testRegionService() throws Exception {
 
 	private static void _addData() throws Exception {
 		_executeNonQuery(
-			"create table region(id bigint not null primary key, name varchar(255));");
+			"create table region" +
```

Comment:
This looks like a dangerous change made by SF.

Reply:
But I guess it knows what it is doing?

Reply:
This file shouldve been ignored for SF. Is this rebased from latest?

Reply:
https://github.com/liferay/liferay-blade-samples/blob/7.1/liferay-workspace/apps/service-builder/jndi/jndi-service/source-formatter.properties